install_files/ansible-base/roles/grsecurity/tasks/apply_grsec_lock.yml

```diff
@@ -12,14 +12,14 @@
     - reboot
 
   # If grsec wasn't already running, wait for the host to come back up
-  # after rebooting. The Ansible variable {{ ansible_ssh_host }} refers
+  # after rebooting. The Ansible variable {{ ansible_host }} refers
   # to the host just rebooted, and this syntax will work in production,
   # vagrant/virtualbox, and snap-ci environments.
 - name: Wait for server to come back.
   local_action:
     module: wait_for
-      host={{ ansible_ssh_host }}
-      port={{ ansible_ssh_port }}
+      host={{ ansible_host }}
+      port={{ ansible_port | default('22') }}
```

Comment:
Careful: while Ansible 2 adds support for `ansible_host`, it doesn't remove support for `ansible_ssh_host`; the SecureDrop production inventory file (`install_files/ansible-base/inventory`) still uses `ansible_ssh_host`, so we should continue to support it. 

See example here of how we can support both: https://github.com/freedomofpress/ansible-role-grsecurity/blob/7cdd39b3afae1d09797467e55f82ba214ae0dbe6/roles/install-grsec-kernel/tasks/validate_install.yml#L16-L23 Although I'd prefer that the default logic were reversed, so that the newer var takes precedence, i.e.

```
ansible_host|default(ansible_ssh_host|default(inventory_hostname))
```

instead of:

```
ansible_ssh_host|default(ansible_host|default(inventory_hostname))
```